app/databases/show/operations/route.js

```diff
@@ -1,14 +1,19 @@
 import Ember from 'ember';
+import Paginated from "diesel/mixins/routes/paginated";
 
-export default Ember.Route.extend({
-  model: function(){
+export default Ember.Route.extend(Paginated, {
+  model: function(params, transition){
     var db = this.modelFor('databases.show');
+    var page = parseInt(transition.queryParams.page || 1, 10);
```

Comment:
I understood that query params are always coerced to the same type as their original value (in this case `1` in the paginated mixin).

Also I believe the query param is on `params`, and accessing the transition is unnecessary. I can confirm.